toolchain/check/context.cpp

```diff
@@ -679,8 +681,7 @@ auto Context::LookupQualifiedName(SemIR::LocId loc_id, SemIR::NameId name_id,
     }
 
     return {.specific_id = SemIR::SpecificId::Invalid,
-            .inst_id = SemIR::ErrorInst::SingletonInstId,
-            .is_poisoned = result.is_poisoned};
```

Comment:
Why does this change from from forwarding poisoned state to just erroring? Is it just that it doesn't matter when the inst_id is ErrorInst?

(not asking for a change, just looking to make sure I understand correctly)

Reply:
I believe that currently we cannot have an error and poisoning together. I could be mistaken, but I do know that none of our tests trigger that.
IIUC, an error only happens when the class, interface or package scope have an error, and a name cannot be poisoned there. I've tried triggering this, but couldn't.